components/ide/jetbrains/image/status/main.go

```diff
@@ -254,6 +258,20 @@ func remoteDevServerCmd(args []string) *exec.Cmd {
 	return cmd
 }
 
+func configureXmx(alias string) error {
+	xmx := os.Getenv(strings.ToUpper(alias) + "_XMX")
+	if xmx == "" {
+		return nil
+	}
+	launcherPath := "/ide-desktop/backend/plugins/remote-dev-server/bin/launcher.sh"
+	content, err := ioutil.ReadFile(launcherPath)
+	if err != nil {
+		return err
+	}
+	newContent := strings.Replace(string(content), "2048m", xmx, 1)
```

Comment:
Should we change `2048m` to `-Xmx2048m`? It's like a magic number here if you not access `launcher.sh` file

Reply:
And add this one https://github.com/gitpod-io/gitpod/blob/a4f50fa74f24c6abe1fa04a6a9e1ea6d0fee3c62/components/ide/jetbrains/image/startup.sh#L13